packages/dropdowns/src/Menu/Menu.js

```diff
@@ -49,10 +61,17 @@ const Menu = props => {
 
   return (
     <MenuContext.Provider value={{ itemIndexRef }}>
-      <Popper placement={popperPlacement} modifiers={popperModifiers} eventsEnabled={eventsEnabled}>
+      <Popper
+        placement={popperPlacement}
+        modifiers={popperModifiers}
+        // Disable position updating on scroll events while menu is closed
+        eventsEnabled={isOpen ? eventsEnabled : false}
```

Comment:
```suggestion
        eventsEnabled={isOpen && eventsEnabled}
```

Notice this statement is not exactly the same, as it might return `undefined`, but if eventsEnabled is `Proptype.bool` it should be fine.